tests/FakerDotNet.Tests/Fakers/TeamFakerTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using FakeItEasy;
+using FakerDotNet.Data;
+using FakerDotNet.Fakers;
+using NUnit.Framework;
+
+namespace FakerDotNet.Tests.Fakers
+{
+    [TestFixture]
+    [Parallelizable]
+    public class TeamFakerTests
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            _fakerContainer = A.Fake<IFakerContainer>();
+            _teamFaker = new TeamFaker(_fakerContainer);
+        }
+
+        private IFakerContainer _fakerContainer;
+        private TeamFaker _teamFaker;
+
+        [Test]
+        public void Creature_returns_a_creature()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(TeamData.Creatures))
+                .Returns("buffalo");
+
+            Assert.AreEqual("buffalo", _teamFaker.Creature());
+        }
+
+        [Test]
+        public void Name_returns_a_name()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(TeamData.Creatures))
+                .Returns("buffalo");
+
```

Comment:
Can we remove this bit of whitespace, so that it looks the same as the other test classes?

```suggestion
```